src/pages/ProductsPage/Products.tsx

```diff
@@ -1,37 +1,40 @@
 import "./products.scss";
 import React, { useState, useEffect } from "react";
 import Product from "../../components/ui/productCard/ProductCard";
+import Loader from "../../components/ui/loader/Loader";
 import { BreedType } from "../../types";
 
 export default function Products() {
-  const [jsonInfo, setInfo] = useState<BreedType[]>([]);
-  const [errorMessage, setError] = useState<string>("");
-  const max = 21;
+  const [breed, setBreed] = useState<BreedType[]>([]);
```

Comment:
this variable should be called in plural form breeds instead of breed since it's array of items